### ISSUE TITLE
Binding IKEA Remote does not work

### ISSUE THREAD

**Issue by philrich** [Koenkk/zigbee2mqtt#16975]
### What happened?

Binding a IKEA Remote (E1524/E1810) to a IKEA Lamp (LED1624G9) returns success but the Lamp does not react to button presses. The Remote Firmware is 2.3.014, so I created a group first and put the Lamp into it. Then I removed the default_group_binding from the remote and added a binding to my created group for the LevelCtrl & OnOff clusters. Now, when I press a button on the remote the light does not react.
I repaired the remote multiple times, tried to create the bindings via frontend and MQTT.
As I am in the process of migrating from Deconz (ConBee II) to Zigbee2MQTT (Sonoff Zigbee Dongle-E) I also tried to pair the Light/Remote to Deconz which works flawless.

### What did you expect to happen?

Remote should be bound to Lamp-Group. Light should respond to Remote.

### How to reproduce it (minimal and precise)

 - Pair IKEA Remote Lamp with old group aware FW
 - Pair IKEA Lamp
 - Create Group
 - Add Lamp to Group
 - Bind Remote to Group
 - Light does not react to Remote

### Zigbee2MQTT version

1.30.2

### Adapter firmware version

6.10.3.0 build 297

### Adapter

Sonoff ZBDongle-E USB Zigbee 3.0 USB Dongle Plus

### Debug log

(No debug log atm but Database Entry of relevant Devices)

```
{"id":86,"type":"EndDevice","ieeeAddr":"0x90fd9ffffe58e8db","nwkAddr":47639,"manufId":4476,"manufName":"IKEA of Sweden","powerSource":"Battery","modelId":"TRADFRI remote control","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":2080,"inClusterList":[0,1,3,32,4096,64636],"outClusterList":[3,4,6,8,25,4096],"clusters":{"genBasic":{"attributes":{"modelId":"TRADFRI remote control","manufacturerName":"IKEA of Sweden","powerSource":3,"zclVersion":3,"appVersion":33,"stackVersion":98,"hwVersion":1,"dateCode":"20190708","swBuildId":"2.3.014"}},"genPollCtrl":{"attributes":{"checkinInterval":172800}},"genPowerCfg":{"attributes":{"batteryPercentageRemaining":87}}},"binds":[{"cluster":32,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1},{"cluster":1,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1},{"cluster":6,"groupID":7,"type":"group"},{"cluster":8,"groupID":7,"type":"group"}],"configuredReportings":[{"cluster":1,"attrId":33,"minRepIntval":3600,"maxRepIntval":62000,"repChange":0}],"meta":{}}},"appVersion":33,"stackVersion":98,"hwVersion":1,"dateCode":"20190708","swBuildId":"2.3.014","zclVersion":3,"interviewCompleted":true,"meta":{"configured":2040954255},"lastSeen":1678357956607,"defaultSendRequestWhen":"active"}
{"id":87,"type":"Router","ieeeAddr":"0x000d6ffffe26033b","nwkAddr":4102,"manufId":4476,"manufName":"IKEA of Sweden","powerSource":"Mains (single phase)","modelId":"TRADFRI bulb E27 CWS opal 600lm","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":258,"inClusterList":[0,3,4,5,6,8,768,4096,64636],"outClusterList":[25,4096],"clusters":{"genBasic":{"attributes":{"modelId":"TRADFRI bulb E27 CWS opal 600lm","manufacturerName":"IKEA of Sweden","powerSource":1,"zclVersion":3,"appVersion":33,"stackVersion":98,"hwVersion":1,"dateCode":"20211025","swBuildId":"2.3.086"}},"lightingColorCtrl":{"attributes":{"colorCapabilities":15}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1},{"cluster":8,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":3600,"repChange":0},{"cluster":8,"attrId":0,"minRepIntval":5,"maxRepIntval":3600,"repChange":1}],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":33,"stackVersion":98,"hwVersion":1,"dateCode":"20211025","swBuildId":"2.3.086","zclVersion":3,"interviewCompleted":true,"meta":{"configured":88764544},"lastSeen":1678316025212,"defaultSendRequestWhen":"immediate"}
{"id":88,"type":"Group","groupID":7,"members":[{"deviceIeeeAddr":"0x000d6ffffe26033b","endpointID":1}],"meta":{}}
```

**Comment by philrich** [comment]
I tried with a second E1524/E1810 Remote which is on Firmware 24.4.5. As per the docs this Remote should be bound to the device directly, which I did. This worked and I am able to controll the Lamp with the Remote.
So in my experience
  - Binding directly to Device with new Firmware 24.4.5 -> works
  - Binding via Group with old Firmware 2.3.014 -> does not work
I have to Light Groups with 8 Lamps each so it would really be important to me to controll these lights via Groups.

Here is the relevant database Entry of the Remote with the new Firmware
```
{"id":5,"type":"EndDevice","ieeeAddr":"0xd0cf5efffef8159f","nwkAddr":33911,"manufId":4476,"manufName":"IKEA of Sweden","powerSource":"Battery","modelId":"TRADFRI remote control","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":2080,"inClusterList":[0,1,3,32,4096,64599,64636],"outClusterList":[3,4,5,6,8,25,4096],"clusters":{"genBasic":{"attributes":{"modelId":"TRADFRI remote control","manufacturerName":"IKEA of Sweden","powerSource":3,"zclVersion":8,"appVersion":36,"stackVersion":106,"hwVersion":1,"dateCode":"20220928","swBuildId":"24.4.5"}},"genPollCtrl":{"attributes":{"checkinInterval":172800}},"genPowerCfg":{"attributes":{"batteryPercentageRemaining":130}}},"binds":[{"cluster":32,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1},{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1},{"cluster":1,"type":"endpoint","deviceIeeeAddress":"0xdc8e95fffe258696","endpointID":1},{"cluster":5,"type":"endpoint","deviceIeeeAddress":"0x000d6ffffe26033b","endpointID":1},{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x000d6ffffe26033b","endpointID":1},{"cluster":8,"type":"endpoint","deviceIeeeAddress":"0x000d6ffffe26033b","endpointID":1}],"configuredReportings":[{"cluster":1,"attrId":33,"minRepIntval":3600,"maxRepIntval":62000,"repChange":0}],"meta":{}}},"appVersion":36,"stackVersion":106,"hwVersion":1,"dateCode":"20220928","swBuildId":"24.4.5","zclVersion":8,"interviewCompleted":true,"meta":{"configured":2040954255},"lastSeen":1678388745198,"defaultSendRequestWhen":"active"}
```

**Comment by philrich** [comment]
I also tried with Zigbee2Mqtt-Edge (1.30.2-dev commit: 347c023). Unfortunately it did not work either.

Here ist the full `database.db` and the relevant debug log:
[database.db.log](https://github.com/Koenkk/zigbee2mqtt/files/10999842/database.db.log)
[z2m_ikea_light_remote_bind.log](https://github.com/Koenkk/zigbee2mqtt/files/10999843/z2m_ikea_light_remote_bind.log)

Either I do something completely wrong or group bindings are broken with the "old" IKEA firmware. Or ist it the Stick? (Sonoff Dongle-E).

**Comment by timdonovanuk** [comment]
Not working for me with:

- [Ikea remote ](https://www.zigbee2mqtt.io/devices/E1524_E1810.html#ikea-e1524%252Fe1810)
- Firmware version 1.2.214 (I will update and report back)
- Zigbee2MQTT version 1.30.2
- Coordinator zStack3x0 on 20220219
- Binding to an Ikea bulb

![image](https://user-images.githubusercontent.com/8156439/228542444-d74bd8d1-4cdf-40f9-8245-ce980d0ca110.png)

**Comment by timdonovanuk** [comment]
> > * Binding to an Ikea bulb
> >   Ah I just read the docs, it definitely does not work on old firmware. Will report back...
> 
> Can you check if it works for you with the old firmware if you add the bulb to a group and bind the remote to this group (as stated in the docs)?

So I actually tried a group first - no go. Then I tried a bulb direct - also no go.

Updating the firmware but it's a 2 hour ETA..

**Comment by jaimemrjm** [comment]
Updating the firmware, after some hours, and binding the bulb device, works for me.

**Comment by philrich** [comment]
Ok, so other users have this problem too.
Upgrading Firmware fixes the Problem but i had a reason not to upgrade. I want to control a group of 8 lights with the remote. Even with direct device binding this ist not possible as there is another issue which limits the maximum possible light which can be bound to one remote to 7 (https://github.com/Koenkk/zigbee2mqtt/issues/16198). This was tested with the On/Off Switch. I also tested with a Styrbar and this remote allows even less bound devices (I'm not sure but I think it was 3).

As a workaround I installed Switch Manager (https://github.com/Sian-Lee-SA/Home-Assistant-Switch-Manager) and defined rules to control the light group with Zigbee2MQTT. But of course this is not what binding is meant for. Remotes should work even if Home Assistant is down.

I own a Zigbee Sniffer so I could upload a Sniff of the binding process if that would help.
Today I tested again with the current Edge Version but it did not work.
